langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicMessageChatModel.java

```diff
@@ -53,47 +59,164 @@ protected Map<String, Object> getRequestParameters(String prompt) {
         parameters.put("anthropic_version", anthropicVersion);
         return parameters;
     }
-    
+
     @Override
     public Response<AiMessage> generate(List<ChatMessage> messages) {
+        return generate(messages, (ToolSpecification) null);
+    }
+
+    @Override
+    public Response<AiMessage> generate(List<ChatMessage> messages, ToolSpecification toolSpecification) {
```

Comment:
This method is **_forcing_**  the LLM to call this specific tool, is this functionality supported? If not, I would keep it unimplemented

Reply:
Yes, like in the class OpenAiChatModel.
![image](https://github.com/user-attachments/assets/2672f844-3973-409f-995c-4f452b9ce31b)

Reply:
I'll implement tool_choice too. I forget.

Reply:
Done.